cip-0008.md

```diff
@@ -0,0 +1,33 @@
+<pre>
+  CIP: 8
+  Title: CIP Implementation Bounties
+  Author: Robby Dermody <robbyd@counterparty.io>
+  Status: Draft
+  Type: Process
+  Created: 2017-02-23
+</pre>
+
+## CIP Implementation Bounties ##
+
+Bounties raised from the Counterparty community can be a useful tool to promote implementation of specific CIPs, via raising the funds sometimes necessary to compensate development time. To help facilitate this, the Counterparty Foundation may appoint one or more CIP Custodians, who shall be in charge of handling funds for currently open CIP bounty efforts. CIP custodians shall serve for an indeterminate amount of time, and may be freely appointed and removed by the Counterparty Foundation members via a majority vote. Imposed limitations are as follows:
+
+* An individual serving as a CIP custodian may not serve as a Counterparty Foundation board member at the same time.
+* An individual may not serve as both a CIP author and CIP custodian for any specific proposal.
+
+Once a CIP is in the draft stage, the CIP's author may choose to contact a CIP custodian, who will provision a donation address for that bounty under the sole control of the CIP custodian. A separate private key/address shall be used to handle the funds for each bounty independently. If multiple CIP custodians are serving, they may choose to jointly secure funds for a specific bounty via a multi-signature scheme, but such a scheme is not required. All funds gathered shall stay at the address to which they were sent while under the control of a CIP custodian.
+
+Before a CIP custodian may provision a donation address for a specific CIP, that CIP must be written to include a section entitled "Milestones", which shall include the following:
+
+* The fundraising goal, being a total amount (expressed in BTC, XCP or USD) that must be raised in order to consider fundraising for the CIP complete. (Contributions over and above this amount will be refunded to their senders by the CIP custodian.)  
+* Two or more discrete milestones, and for each list specific criteria which must be met in order to consider the milestone complete.
```

Comment:
I think a CIP could conceivably have a single milestone.  Do you think we need to enforce a minimum of two?